composer/profiler/utils.py

```diff
@@ -93,5 +93,5 @@ def export_memory_timeline_html(prof: TorchProfile,
 
         with open(path, 'w') as f:
             f.write(html)
-    log.debug('Memory timeline exported to', path, '.')
+    log.debug('Memory timeline exported to %s.', path)
```

Comment:
```suggestion
    log.debug(f'Memory timeline exported to {path}.')
```

Reply:
@mvpatel2000 This is actually slower as the fstring will be constructed whether or not the log is in debug mode.  See https://docs.astral.sh/ruff/rules/logging-f-string/ , https://pypi.org/project/flake8-logging-format/ .